target/linux/ramips/image/mt7621.mk

```diff
@@ -2162,6 +2162,20 @@ define Device/netgear_wndr3700-v5
 endef
 TARGET_DEVICES += netgear_wndr3700-v5
 
+define Device/netis_n6
+  $(Device/dsa-migration)
+  $(Device/nand)
+  DEVICE_VENDOR := netis
+  DEVICE_MODEL := N6
+  IMAGE_SIZE := 129408k
```

Comment:
I think that you have forgot to recalculate image size. Something like:
max image size=128MiB - 8MiB(nmbm area) - 1.5MiB (boot+config+factory) - 4MiB (kernel)

Reply:
Thanks for the finding. We should remove IMAGE_SIZE completely. It isn't used when checking any of the images we create.